### PR TITLE
[Shop] Sort product by price

### PR DESCRIPTION
**Pull request by tuka217**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets |-
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Context/Setup/ProductContext.php [hunk 1]
```diff
@@ -540,6 +540,15 @@ public function thisProductHasAnImageWithACode(ProductInterface $product, $image
     }
 
     /**
+     * @Given /^(this product)'s price is "([^"]+)"$/
+     */
+    public function thisProductPriceIs(ProductInterface $product, $price)
+    {
+        $product->getFirstVariant()->setPrice($this->getPriceFromString($price));
```
Comment (michalmarcinkowski) [inline src/Sylius/Behat/Context/Setup/ProductContext.php]: `getFirstVariant()` should not be used anymore, we should always use `defaultVariantResolver` as we do e.g. [here](https://github.com/tuka217/Sylius/blob/fdef0bf31a94eb6b3b34c4394101870fe72af188/src/Sylius/Behat/Context/Setup/ProductContext.php#L497).

PS. We should remove `getFirstVariant` method from `Product` in a separate PR.

**Comment by michalmarcinkowski** [comment]
Thanks Ania! 👍

**Comment by Zales0123** [comment]
Something is wrong with the build, it should fail :/
zrzut ekranu 2016-10-13 o 12 36 51

**Comment by michalmarcinkowski** [comment]
Probably "rerun" overrides the result of the build, but don't take into account that there were undefined steps... :/

/cc @pamil

**Pull request merged**